files/pt-br/conflicting/web/api/animationevent/animationevent/index.html

```diff
@@ -72,59 +72,7 @@ <h2 id="Especificações">Especificações</h2>
 
 <h2 id="Browser_compatibility">Compatibilidade com navegadores</h2>
 
-<p>{{ CompatibilityTable }}</p>
```

Comment:
Curious about the intent of the `conflicting` folder. I believe files in a folder called `orphaned` is meant to be deleted as they no longer exist in the English content. What is the utility of the `conflicting` folder? Is this documented somewhere? Thanks!

Reply:
Files in the `conflicting` folder are usually moved there by the `sync-translated-content` workflow, which tried to move a file to a "new" slug but a page with that slug already exists (this often occurs when a group of pages are deleted on mdn/content and are all redirected to the same page). 

I believe there isn't really documentation on conflicting/orphaned, but I [wrote a gist for this](https://gist.github.com/cw118/00424f18db4476e8b76dd31842ed9bad) some time ago :)

Reply:
Thank you for the feedback, @cw118. Might be worth adding a more detailed explanation to the README or CONTRIBUTING docs?

Reply:
@cw118 are you going to open a PR for this? You can also push to my branch if you want.

Reply:
@schalkneethling Sorry for the late reply, recent weeks have been busy for me :) I'd be happy to work on this, probably in another PR as I think conflicting/orphaned guidelines are not quite in the scope of this one.

Reply:
I 100% agree. I assume you will take care of this one as well? https://github.com/mdn/translated-content/pull/8973#discussion_r988342233